src/scale.jl

```diff
@@ -599,23 +599,18 @@ end
 immutable ContinuousColorScale <: Gadfly.ScaleElement
     # A function of the form f(p) where 0 <= p <= 1, that returns a color.
     f::Function
+    trans::ContinuousScaleTransform
 
     minvalue
     maxvalue
 
-    function ContinuousColorScale(f::Function; minvalue=nothing, maxvalue=nothing)
-        new(f, minvalue, maxvalue)
+    function ContinuousColorScale(f::Function, trans::ContinuousScaleTransform; minvalue=nothing, maxvalue=nothing)
```

Comment:
I think the problem is on this line.  Since the `trans` argument did not exist before, and a default value is not provided, it is breaking any existing code that simply calls `ContinuousColorScale(f)`.  Try turning it to `trans::ContinuousScaleTransform = identity_transform`?